### ISSUE TITLE
Make use of new job summary markdown feature for actions

### ISSUE THREAD

**Issue by SebastianSchuetze** [dorny/test-reporter#192]
For GitHub the following was released:

> To summarize the result of a job, users can generate Markdown and publish the contents as a job summary. For example, after running tests with GitHub Actions, a summary can provide an overview of passed, failed, or skipped tests, potentially reducing the need to review the full log output. For more information, see "[Workflow commands for GitHub Actions](https://docs.github.com/en/enterprise-server@3.6/actions/using-workflows/workflow-commands-for-github-actions#adding-a-job-summary)."
https://docs.github.com/en/enterprise-server@3.6/admin/release-notes

It would be really grad if this extension would make use of it as soon as possible. I think it makes your action much more powerfull and easier to use in many places. I am using your action for our Azure DevOps PowerShell modules VSTeams to make summaries. But I had problems with permissions when running on pull requests (not running as a user but technical user which has no rights to do what your actions is doing. But this makes it independent from that).

Here are the docs for this:
https://docs.github.com/en/enterprise-server@3.6/actions/using-workflows/workflow-commands-for-github-actions#adding-a-job-summary

It was literally released a few hours ago.

**Comment by SebastianSchuetze** [comment]
Seems like no maintance is done here anymore.

**Comment by dorny** [comment]
Yep, in the past year or more, the maintenance was close to zero. I'm trying to avoid total burnout and I had to skip all non-essential worktime (i.e. paid job & family-related stuff). I stopped reacting to issues a while ago, but seeing this open for a year makes me sad. I still hope I can get back to this at some point, so let's keep it open.

**Comment by SebastianSchuetze** [comment]
It is fine. Just stated the obvious. I know what it means. Sometimes it is also fine to cut things. That means also more focus on other.

**Comment by owns** [comment]
I believe this issue can now be closed with release v2 `use-actions-summary: 'true'` thanks to #463
